ytfzf

```diff
@@ -818,7 +818,7 @@ get_search_query () {
 			printf "$search_prompt"
 			read -r search_query
 		fi
-		[ -z "$search_query" ] && clean_up && exit 0
+		[ -z "$search_query" ] && (clean_up; exit 0)
```

Comment:
I don't know if this is intensional but `()` will run the code in a sub shell (won;t make much of a difference here). Use `{}` for grouping commands with out subshell

Reply:
() makes exit only happen if [ -z $search_query Is true

Reply:
if you do `(exit 0)` it will not exit the script. It will only exit the subshell

Reply:
fixed